src/main/java/core/machines/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.machines;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started work");
```

Comment:
The message in the `doWork` method should be more informative. According to the checklist, it should indicate what type of `Machine` is working. Consider specifying that this is a `Truck` doing some specific work, for example, 'Truck is hauling cargo'.